### PR TITLE
[PT-BR] Babel with TypeScript.md

### PR DESCRIPTION
**Pull request by LuizDoPc**
This PR contains the Brazilian Portuguese translation of the file Babel with TypeScript.md

@khaosdoctor, @alvarocamillont, @danilofuchs, @orta

### CODE REVIEW

File: packages/documentation/copy/pt/tutorials/Babel with TypeScript.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
+title: Using Babel with TypeScript
```
Comment (khaosdoctor) [inline packages/documentation/copy/pt/tutorials/Babel with TypeScript.md]:
```suggestion
title: Usando Babel com TypeScript
```

File: packages/documentation/copy/pt/tutorials/Babel with TypeScript.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+layout: docs
+permalink: /pt/docs/handbook/babel-with-typescript.html
+oneline: Como criar um projeto híbrido com Babel + TypeScript
+translatable: true
+---
+
+## Babel vs `tsc` para TypeScript
+
+Desenvolvendo um projeto JavaScript moderno, você pode se perguntar qual a forma correta de converter arquivos do TypeScript para o JavaScript.
```
Comment (khaosdoctor) [inline packages/documentation/copy/pt/tutorials/Babel with TypeScript.md]:
```suggestion
Ao desenvolver um projeto JavaScript moderno, você pode se perguntar qual a forma correta de converter arquivos do TypeScript para o JavaScript.
```

File: packages/documentation/copy/pt/tutorials/Babel with TypeScript.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+Várias vezes a resposta é _"depende"_, ou _"alguém pode ter decidido por você"_ dependendo do projeto. Se você está construindo seu projeto com algum framework existente como [tsdx](https://tsdx.io), [Angular](https://angular.io/), [NestJS](https://nestjs.com/) ou qualquer outro framework mencionado na seção [Getting Started](/docs/home) então essa decisão é sua.
```
Comment (khaosdoctor) [inline packages/documentation/copy/pt/tutorials/Babel with TypeScript.md]:
```suggestion
Várias vezes a resposta é _"depende"_, ou _"alguém pode ter decidido por você"_ dependendo do projeto. Se você está construindo seu projeto com algum framework existente como [tsdx](https://tsdx.io), [Angular](https://angular.io/), [NestJS](https://nestjs.com/) ou qualquer outro framework mencionado na seção [Começando com TypeScript](/docs/home) então essa decisão é sua.
```

**Review by khaosdoctor — CHANGES_REQUESTED** [review]
Small changes

**Comment by LuizDoPc** [comment]
@khaosdoctor done

**Review by khaosdoctor — APPROVED** [review]
Nice!

**Comment by khaosdoctor** [comment]
lgtm

**Pull request merged**